pnpm-workspace.yaml

```diff
@@ -9,3 +9,5 @@ packages:
 overrides:
   "cross-spawn@>=7.0.0 <7.0.5": "^7.0.5"
   rollup: 4.49.0 # Regression in 4.50.0 https://github.com/rollup/rollup/issues/6099
+  "@alloy-js/core": 0.21.0-dev.8
+  "@alloy-js/csharp": 0.21.0-dev.19
```

Comment:
How can I upgrade to use dev alloy-js libraries in this repo? I tried to upgrade all the packages including alloy-js/core and alloy-js/csharp to the dev version but the build would still fail complaining about conflictions with alloy-js/core@0.20.0 which i guess is because other alloy libs are at 0.20.0 version. Then I change to add the override here which fixes the build, but the typespec-azure in pipeline would still complain about the 0.20.0...

Reply:
we try not to switch to dev version in main, what we can do is create a feature branch with any pending things and use the pr link until alloy is upgraded